hazelcast/src/main/java/com/hazelcast/cache/impl/AbstractCacheRecordStore.java

```diff
@@ -619,6 +619,10 @@ protected R updateRecord(Data key, R record, Object value, int completionId, Str
         }
     }
 
+    protected void updateRecordValue(R record, Object recordValue) {
```

Comment:
so enterprise also handles HB for heap based cache also when hot-restart is enabled?